### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by wilmhit**
Unit test not applicable

### CODE REVIEW

File: easywarehouse/easywarehouse/urls.py [hunk 1]
```diff
@@ -13,9 +13,15 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
-from django.urls import path
+from django.contrib import admin, auth
+from django.contrib.auth.views import LoginView, LogoutView
+from django.urls import include, path
+from products import views
```
Comment (songofroland) [inline easywarehouse/easywarehouse/urls.py:19]: Please always run `make format` before submitting changes

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+__pycache__
```
Comment (songofroland) [inline .gitignore:1]: 👍

File: easywarehouse/products/templates/registration/login.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        {{ form.username }}
+    </label><br />
+    <label>
+        Password
+        {{ form.password }}
+    </label><br/>
+    {{ form.errors }}
+    <input type="submit" />
+</form>
+</html>
```
Comment (songofroland) [inline easywarehouse/products/templates/registration/login.html:16]: EOF

File: easywarehouse/products/views.py [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
 # Create your views here.
+
+
+def index(request):
+    return HttpResponse('<h1>Hello, this is index page</h1> <a href="/login">login</a>')
+
+
+@login_required
+def dashboard(request):
+    return HttpResponse("Hello emoployee. You are logged in")
```
Comment (songofroland) [inline easywarehouse/products/views.py:14]: Well done, could you please add a task related to login/logout styles?

**Review by songofroland — APPROVED** [review]

File: easywarehouse/easywarehouse/urls.py (hunk 1 above)
Comment (wilmhit) [reply easywarehouse/easywarehouse/urls.py:19]: did run

**Pull request merged**